MAINTAINERS

```diff
@@ -179,6 +179,7 @@ S:	Supported
 F:	providers/vmw_pvrdma/
 
 PYVERBS
-M:	Noa Osherovich <noaos@mellanox.com>
+M:	Edward Srouji <edwards@mellanox.com>
+H:	Noa Osherovich <noaos@mellanox.com>
```

Comment:
there is no need in this line, because Noa's email is not active.

Thanks

Reply:
Removed